src/ray/raylet/node_manager.cc

```diff
@@ -1297,9 +1296,9 @@ void NodeManager::HandleDisconnectedActor(const ActorID &actor_id, bool was_loca
   RAY_CHECK(actor_entry != actor_registry_.end());
   auto &actor_registration = actor_entry->second;
   auto remainingRestarts = actor_registration.GetRemainingRestarts();
-  RAY_LOG(DEBUG) << "The actor with ID " << actor_id << " died "
-                 << (intentional_disconnect ? "intentionally" : "unintentionally")
-                 << ", remaining restarts = " << remainingRestarts;
+  RAY_LOG(WARNING) << "The actor with ID " << actor_id << " died "
```

Comment:
We should also make it INFO if intentional disconnect is true.

Reply:
fixed